presto-main/src/main/java/com/facebook/presto/operator/aggregation/state/StateCompiler.java

```diff
@@ -741,4 +764,26 @@ else if (initialValue instanceof Boolean) {
             }
         }
     }
+
+    public static final class StateClass<T>
```

Comment:
Probably not. I added this when I used to generate interface and I want the function that generates interface to return both. Now they are separate anyway.